Interfaces/Preferences.xib

```diff
@@ -1992,11 +1992,11 @@ Warning: If Sequel Ace and the server do not have at least one cipher suite in c
                         <constraint firstAttribute="height" constant="16" id="UHj-Wq-xiC"/>
                         <constraint firstAttribute="width" constant="11" id="nHg-43-jBJ"/>
                     </constraints>
-                    <stepperCell key="cell" controlSize="small" continuous="YES" enabled="NO" alignment="left" increment="0.10000000000000001" minValue="0.5" maxValue="5" doubleValue="1" id="1526">
```

Comment:
Should we still set a minValue of 0 so it doesn't go negative?

Reply:
Looks like this may be handled later actually?

Reply:
yeah, can't go negative.